cts-java/src/test/java/org/ga4gh/cts/api/variants/VariantsMethodsEndpointAliveIT.java

```diff
@@ -0,0 +1,71 @@
+package org.ga4gh.cts.api.variants;
+
+import org.ga4gh.ctk.CtkLogs;
+import org.ga4gh.ctk.transport.GAWrapperException;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.ga4gh.ctk.transport.protocols.Client;
+import org.ga4gh.cts.api.Utils;
+import org.ga4gh.methods.SearchCallSetsRequest;
+import org.ga4gh.methods.SearchVariantsRequest;
+import org.ga4gh.methods.SearchVariantsResponse;
+import org.ga4gh.models.CallSet;
+import org.ga4gh.models.Variant;
+import org.ga4gh.models.VariantSet;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.net.HttpURLConnection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Verify basic operations on variants, variantsets, and callsets.
+ */
+@Category(VariantsTests.class)
+public class VariantsMethodsEndpointAliveIT implements CtkLogs {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     * Check that searching {@link Variant}s with a nonexistent reference name returns
```

Comment:
The agreed (but not documented?) behavior, as I understand it, is that nonexistent IDs return a 404; nonexistent (non-matching) names filter by the name - and in this case, return an empty list.  I don't have a pointer to the discussion handy, unfortunately.